### ISSUE TITLE
[Bug] Setting biome edge to zero will make impossible to load world

### ISSUE THREAD

**Issue by paulevsGitch** [quiqueck/BCLib#7]
### What happened?

If the biome edge value will be zero the world will be impossible to load. Found this in 1.18.2, probably can affect 1.19.

![2022-06-17_20 42 34](https://user-images.githubusercontent.com/17707380/174350758-958a7228-a789-4ca6-a2ae-87b575506685.png)

How to recreate:
```json
"nether_mushroom_forest": {
      "generation_chance [default: 1.0]": 1.0,
      "edge_size [default: 6]": 0,
      "fog_density [default: 1.0]": 1.0,
      "vertical [default: false]": false,
      "terrain_height [default: 0.1]": 0.1
},
```

### BCLib

1.4.7

### Fabric API

0.53.4

### Fabric Loader

0.14.8

### Minecraft

1.18.2

### Relevant log output

```shell
java.util.NoSuchElementException: No value present
	at java.util.Optional.orElseThrow(Optional.java:377) ~[?:?]
	at ru.bclib.world.biomes.BCLBiome.updateActualBiomes(BCLBiome.java:202) ~[bclib-1.4.7.jar:?]
	at ru.bclib.world.biomes.BCLBiome.updateActualBiomes(BCLBiome.java:199) ~[bclib-1.4.7.jar:?]
	at ru.bclib.world.generator.BCLibNetherBiomeSource.lambda$new$4(BCLibNetherBiomeSource.java:93) ~[bclib-1.4.7.jar:?]
	at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?]
	at ru.bclib.world.generator.BCLibNetherBiomeSource.<init>(BCLibNetherBiomeSource.java:93) ~[bclib-1.4.7.jar:?]
```

### Other Mods

```shell
BN, BE, BYG, Terrablender
```

**Comment by paulevsGitch** [comment]
Same will happen if Edge biome entry will have generation chance 0.0

**Comment by quiqueck** [comment]
Should be fixed for 1.19 in 2.0.7
